dotnet/HtmlFormatterTest/CucumberMessageEnumConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace htmlformatterTest
```

Comment:
Fix namespace and check capitalization: `Cucumber.HtmlFormatterTest` (for all other file too)

Reply:
Done